guides/7-jlp/2-How-JLP-Works.md

```diff
@@ -33,6 +33,12 @@ JLP can also be sold via Jupiter Swap. Simply swap JLP for your desired asset. T
 The best way to purchase or exit JLP is always via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).
```

Comment:
```suggestion
The best way to purchase or exit JLP is always via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).

**Purchasing/Exiting JLP** 

The best way to purchase or exit JLP is via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).

When using Jupiter Swap or programs that integrate Jupiter Swap, the minting and burning of JLP tokens occur automatically. When you swap in and out of JLP, Jupiter Swap will find the best route to mint/burn JLP automatically, or swap through other DEXes if the route offered is better.

:::info
This automatic minting/burning mechanism is unique to Jupiter Swap and programs that route via Jupiter Swap. If you're interacting directly on a different DEX, you will trade JLP at the price offered by the DEX instead of the virtual price of JLP.

These transactions would also not affect the overall JLP supply - only the Jupiter Swap program has the authority to mint and burn JLP tokens.
:::
```